sources/packages/backend/apps/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -81,7 +79,7 @@ export class NoteAESTController extends BaseController {
   @Get("institution/:institutionId")
   async getInstitutionDetails(
     @Param("institutionId", ParseIntPipe) institutionId: number,
-    @Query("noteType", new ParseEnumPipe(NoteType)) noteType: NoteType,
+    @Query("noteType") noteType: NoteType,
```

Comment:
Can you please elaborate on why the `ParseEnumPipe` was removed?

Reply:
Tha "All notes" is not available in the enum NoteType, so had to remove it, or we may need to notetype to extend the Allnotes with the institution and students notetype.

Reply:
A `NoteType` enum can be created at the DTO layer and include the item needed for the controller. No need to remove a security check that was already in place.

Reply:
@guru-aot on click of all notes, is UI adding the query parameter note type? If so then we can make Vue side not to add the query parameter on click of all notes.

Reply:
I guess the note types is not sent on click of all notes. We can talk about it. 

![image](https://user-images.githubusercontent.com/54600590/195946260-76102e95-8d7e-444a-bb25-239248167156.png)

Reply:
@dheepak-aot  you are right, when all notes is selected, undefined is sent. as for the filternotes method, its an optional parameter. We can add all notes in the dto enum and pass in the method, lets discuss.

Reply:
@guru-aot can you please just make clear the plan here? Are you going to leave this one to the tech debit assuming that we will be introducing a query string pipe to accept nullable enums?

Reply:
@andrewsignori-aot yes, we are planning to do this in a tech debt to accept null/ undefined values in the enum validators.